cmd/agentcti/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+import (
+	"github.com/gdamore/tcell/v2"
+	"github.com/rs/zerolog/log"
+	"github.com/sethvargo/go-retry"
+)
+
+import (
+	"github.com/pochtalexa/go-cti-middleware/internal/agent/auth"
+	"github.com/pochtalexa/go-cti-middleware/internal/agent/flags"
+	"github.com/pochtalexa/go-cti-middleware/internal/agent/logger"
+	"github.com/pochtalexa/go-cti-middleware/internal/agent/pgui"
+	"github.com/pochtalexa/go-cti-middleware/internal/agent/storage"
+)
```

Comment:
Нет! Видимо, я был не очень очевиден, когда писал пост в телеграме. Блок - это список импортов, отделенный от остальных двойным переносом строк.
```suggestion
import (
	"context"
	"encoding/json"
	"errors"
	"fmt"
	"net"
	"net/http"
	"os"
	"strings"
	"time"

	"github.com/gdamore/tcell/v2"
	"github.com/rs/zerolog/log"
	"github.com/sethvargo/go-retry"

	"github.com/pochtalexa/go-cti-middleware/internal/agent/auth"
	"github.com/pochtalexa/go-cti-middleware/internal/agent/flags"
	"github.com/pochtalexa/go-cti-middleware/internal/agent/logger"
	"github.com/pochtalexa/go-cti-middleware/internal/agent/pgui"
	"github.com/pochtalexa/go-cti-middleware/internal/agent/storage"
)
```
Как-то так. В принципе, это не блокер, просто хотел внести ясность.